### ISSUE TITLE
TerminalPosition should implements hashCode() and equals()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Valyrox/lanterna#81]
```
TerminalPosition should implements hashCode() and equals()
```

Original issue reported on code.google.com by `segfault...@gmail.com` on 2 Aug 2013 at 8:17

**Comment by GoogleCodeExporter** [comment]
```
Yes, it should
```

Original comment by `mab...@gmail.com` on 3 Aug 2013 at 5:38
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed on default and 2.1.x-branch
```

Original comment by `mab...@gmail.com` on 3 Aug 2013 at 5:43
- Changed state: **Fixed**
